packages/core/src/config.ts

```diff
@@ -612,6 +613,19 @@ const composeFormatConfig = ({
   }
 };
 
+const formatRsbuildPlugin = (): RsbuildPlugin => ({
```

Comment:
Does modify `module.parser.javascript.url` not takes effect?

Reply:
it will affect less url parsing.

Reply:
@SoonIter

Reply:
https://github.com/web-infra-dev/rslib/actions/runs/12115926632/job/33775293755?pr=509